feat(dashboards): Added Data Set section to Custom Dashboards (#4568)
getsentry/sentry-docs: src/docs/product/issues/index.mdx

Learn more about triaging issues and their different states in [Issue States and Triage](/product/issues/states-triage/).

> Early Adopters can also add Issues data to their [Custom Dashboards](/product/dashboards/custom-dashboards/) as widgets through a [Data Set](/product/dashboards/custom-dashboards/#data-set) selector.

Reviewer: ```suggestion
If you're an Early Adopter, you can also add issues data to your [custom dashboards](/product/dashboards/custom-dashboards/) as widgets using the [data set selector](/product/dashboards/custom-dashboards/#data-set).
```

> If you're an Early Adopter, you can also add issues data to your [custom dashboards](/product/dashboards/custom-dashboards/) as widgets using the [data set selector](/product/dashboards/custom-dashboards/#data-set-selection).

Reviewer: ```suggestion
If you're an Early Adopter, you can also add issues data to your [custom dashboards](/product/dashboards/custom-dashboards/), as widgets, using the [data set selector](/product/dashboards/custom-dashboards/#data-set-selection).
```
